ocean_provider/file_types/definitions.py

```diff
@@ -30,11 +29,7 @@ def check_details(self, with_checksum=False):
         raise NotImplementedError
 
     @abstractmethod
-    def build_download_response(
-        self,
-        request,
-        validate_url=True,
-    ):
+    def build_download_response(self, request):
```

Comment:
why has the validate_url parameter disappeared from definitions? I understand in IPFS and Arweave we are not using it, but with regular URL files we would like to keep the option of skipping validation if previously validated.

Reply:
I removed the `is_safe_url()` check from `build_download_response` because it was redundant:

> - Remove `is_safe_url` check from `build_download_response`
>   * When calling download, `is_safe_url()` is called earlier by `check_details()` so calling again in `build_download_response` is redunant.
>   * When calling computeResult, `validate_url=False` so `is_safe_url()` check is skipped anyway.